### PR TITLE
fix(Participant): hide ban option for federated users

### LINKED ISSUE #12955: Ban option shown when removing a federated user
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

Federated users are currently not supported by the API (and its complicated with the proxying anyway)

### PR DESCRIPTION
**Pull request by DorraJaouad**
### ☑️ Resolves

* Fix #12955

### 🖼️ Screenshots / Screencasts

Alice is a federated user

🏚️ After | 🏡 Before
-- | --
image | image

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

**Review by Antreesy — APPROVED** [review]
Apart from test - good to go. Are screenshots correct?)

**Comment by DorraJaouad** [comment]
> Are screenshots correct?)

Flip the reading direction and we are good :p

**Review by Antreesy — APPROVED** [review]

File: src/components/RightSidebar/Participants/Participant.spec.js [hunk 1]
```diff
@@ -713,7 +713,7 @@ describe('Participant.vue', () => {
 				conversation.participantType = PARTICIPANT.TYPE.MODERATOR
```
Comment (nickvergessen) [inline src/components/RightSidebar/Participants/Participant.spec.js:713]: should adjust the `test()` string to avoid confusion

Comment (Antreesy) [reply src/components/RightSidebar/Participants/Participant.spec.js:713]: typo ☝🏽

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**